src/main/resources/META-INF/resources/settings/settings.js

```diff
@@ -1,6 +1,16 @@
 const darkmodeSystemInputElement = document.getElementById('darkmodeSystem');
 const darkmodeOnInputElement = document.getElementById('darkmodeOn');
 const darkmodeOffInputElement = document.getElementById('darkmodeOff');
+const userNameStorageKey = "emtee-user"
+const userNameElement = document.getElementById('userName');
+
+
+// Retrieve the value from localStorage
+
+function displayUserName() {
```

Comment:
Consider more fitting name like `updateDisplayedUserName` since the function needs to be called everytime the username changes.